### ISSUE TITLE
http://schema.geolink.org/1.0/voc/nvs/C17 not accessible

### ISSUE THREAD

**Issue by krisnadhi** [ec-geolink/design#87]
C17.owl is not accessible via its ontology URI: http://schema.geolink.org/1.0/voc/nvs/C17

**Comment by krisnadhi** [comment]
Ah my bad. The URI is wrong.

**Comment by krisnadhi** [comment]
@ashepherd, any idea how to make it accessible?

**Comment by ashepherd** [comment]
hey @krisnadhi , 

If you are calling the URI http://schema.geolink.org/1.0/voc/nvs/C17 from a browser it's going to try to build the HTML version of the OWL file. It looks like a big file, so that might be why it hangs. To get the OWL, you can add an accept header for application/xml+rdf or text/turtle or call it directly like:

http://schema.geolink.org/1.0/voc/nvs/C17.owl

**Comment by krisnadhi** [comment]
Ah indeed. The file is big. Even the LODE service at esseputanto stops loading when given this OWL file.
Is there any way to play around with LODE installation to find a work around this?

**Comment by ashepherd** [comment]
been messing with LODE yesterday. ran it bypassing the proxy, but that failed to render. tried with no reasoning so it could skip using the owlapi Java code which should just apply the stylesheet to the OWL file, and that almost crashed the machine we run it on after running for 4 hours. Tried Parrot overnight, but that is still running too.

Any other tools or thoughts? otherwise, we may have to generate the HTML manually.

**Comment by krisnadhi** [comment]
Yes. I also tried in on my local machine, and it didn't always work. 
Also, the other concern is that when one wants to load main.owl on Protege remotely (retrieving it from schema.geolink.org directly), Protege would fail to load C17.owl because it is too big. 
I looked at this particular vocabulary again, which contains a catalog of platforms, and it seems that quite a lot of them are deprecated. If a platform name/code is deprecated, is it no longer used by any party? 

Let me try if using turtle (instead of rdf/xml) would alleviate the problem.

Btw, your customization of LODE are done on the Java source?

**Comment by ashepherd** [comment]
no customizations needed yet. they had already setup query string parameters to dictate whether owlapi is used or not. but it seems that just applying the XSL stylesheet to the doc in Java was causing the memory issues. we could try applying a trimmed-down version of a XSL stylesheet manually as one option.
